src/polymer/misspelled-behaviors.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Document} from 'polymer-analyzer/lib/model/model';
+
+import stripIndent = require('strip-indent');
+import {Warning, Severity} from 'polymer-analyzer/lib/warning/warning';
+import {stripWhitespace} from '../util';
+import {Rule} from '../rule';
+import {registry} from '../registry';
+
+export class MisspelledBehaviors extends Rule {
+  code = 'misspelled-behaviors';
+  description = stripIndent(`
+      Warns for missing and misspelled behaviors:

```

Comment:
I think `behaviours` is the correct spelling in some parts of the world, which is why it's such a common mistake.

Something like:

```
Warns when the Polymer `behaviors` property is spelled `behaviours`, as Polymer uses the American spelling.
```

I will never hear the end of it from @notwaldorf if I approve a PR that calls the Canadian spelling incorrect